content/en/tools/digital-currencies/activating-token-definitions.md

```diff
@@ -0,0 +1,85 @@
+---
+date: '2023-04-24'
+lastmod: '2023-04-24'
+section_menu: tools
+menu:
+  tools:
+    name: "Activating Token Definitions"
+    weight: 850
+    parent: digital-currencies-tokens-overview
+    identifier: digital-currencies-activating-token-definitions
+    description: "Digital Currencies documentation describing how to activate decommissioned token definitions via the GUI"
+title: "Activating Token Definitions"
```

Comment:
i cant seem to find creating a token definition. creation is the step to defining a token definition. once defined it is set to active

Reply:
Hi Neera, creating token definitions is here:

https://github.com/corda/corda-docs-portal/pull/1322/files#diff-a1208badb8094095cdb96e99cbb78c8a143bfb2ee0bff68e1a9b33cc9a9d4142